web/src/test/resources/com/graphhopper/http/resources/cargo_bike.yml

```diff
@@ -2,7 +2,7 @@
 # profile: bike
 
 # let's assume cargo bikes are e-bikes and we should lower speed to prefer shorter and not faster routes automatically
-max_speed:
+speed:
```

Comment:
Nice! With this branch custom models not only become more powerful but also easier to use :heart:

Reply:
See also the latest commit where I removed the max speed fallback :) 

And do you think we could/should make `multiply with`a bit shorter as it is so frequently used? And is it correct or should it be `multiply by`?

Reply:
> And is it correct or should it be multiply by?

Nice catch! Yes it seems like it should be `multiply by`. So this is already two characters shorter ;).